include/cute/numeric/int.hpp

```diff
@@ -101,4 +103,7 @@ using uint_byte = uint_bit<8*N>;
 template <int N>
 using uint_byte_t = typename uint_byte<N>::type;
 
+template<int Bits, bool Signed>
+struct is_integral<cutlass::integer_subbyte<Bits, Signed>> : true_type {};
```

Comment:
add one for unsigned too?

Reply:
The template covers both since unsigned is `Signed=false`, see [here](https://github.com/NVIDIA/cutlass/blob/56b46e2d13875b46b8f6a03f9f5ac91e2bfdc01a/include/cutlass/integer_subbyte.h#L155C1-L155C44)